AutomatedTesting/Gem/PythonTests/scripting/ScriptEvent_AddRemoveParameter_ActionsSuccessful.py

```diff
@@ -15,8 +26,9 @@ class Tests():
     parameter_removed   = ("Successfully removed parameter",   "Failed to remove parameter")
 # fmt: on
 
+FILE_PATH = os.path.join(paths.projectroot, "ScriptCanvas", "test_file.scriptevent")
```

Comment:
Let's give this a slightly more meaningful name. With a more meaningful name, this may be a candidate for the CONSTS utils, unless it really would be a one-off use.

Reply:
Checked other tests for script events and use the same path. Moving this into the constants file.